### PR TITLE
feat(endpoints): adds baseplate POST endpoint

### PR DESCRIPTION
**Pull request by soph-iest**
*What:** "POST" to /movies now returns "hello world"

**Why:** https://www.notion.so/lob/Endpoint-Configuration-b35722a0321a4f53affc7d2f8eaff652

**Details:** Tested and works!

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.test.*
```
Comment (sjl2) [inline .prettierignore]: nit: is this needed still? Can you add the new line.

File: lib/server.js [hunk 2]
```diff
@@ -12,4 +12,14 @@ const server = new Hapi.Server({
 
 server.connection({ port: Config.PORT });
 
+server.register(
+  [require('hapi-bookshelf-serializer'), require('./plugins/features/movies')],
```
Comment (sjl2) [inline lib/server.js]: super nit: can you have each of these `require`s on their own line?

File: test/models/movie.test.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-trailing-spaces */
```
Comment (sjl2) [inline test/models/movie.test.js]: Discussed previously, but can you remove this?

Comment (soph-iest) [reply test/models/movie.test.js]: Oops, thought I did that; think I may have done something weird with merging that walked it back. Removed now.

**Comment by sjl2** [comment]
As a point of feedback on commit messages, I would strive for a more specific subject. Most changes will involve the endpoint, but in this case, you're working on the `movies` endpoints.

Something like:
```
feat(movies): add POST endpoint base
```

**Pull request merged**